frontend/src/components/mapMarker/parts/MapMarkerCard.tsx

```diff
@@ -1,65 +1,78 @@
 import './MapMarkerCard.css'
 import {MapMarker} from "../../../types/MapMarker.ts";
-import React, {useMemo} from "react";
+import React, {useEffect, useState} from "react";
 import ToolBar from "./ToolBar/ToolBar.tsx";
-import {BsGeoAltFill} from "react-icons/bs";
-import {IconContext} from "react-icons";
+import Draggable, {DraggableData, DraggableEvent} from "react-draggable";
+import MapMarkerIcon from "./MapMarkerIcon.tsx";
 
 
 type MapMarkerCardProps = {
     mapMarker: MapMarker;
-    handleArticleChange: (articleId:string) => void;
     offsetWorldMapFrame: {xOffset:number, yOffset:number};
     isSelected:boolean;
+    isMovable:boolean;
+    handleArticleFrame: () => void;
+    handleMapMarkerUpdate: () => void;
     handleSelectedMapMarkerChange: (mapMarker:MapMarker) => void;
-    handleUpdateMapMarker: () => void;
+    setSelectedMapMarker: (mapMarker:MapMarker) => void;
 }
 
 export default function MapMarkerCard(props: Readonly<MapMarkerCardProps>): React.ReactElement {
-    const mapIconSize={xSize: 80, ySize: 80};
-    const iconContextObj = useMemo(() => ({className: 'mapMarkerIcon'}), []); // value is cached by useMemo
+    const mapMarkerSize={xSize: 80, ySize: 80};
+    const [coordinates, setCoordinates] = useState({xPosition: 0, yPosition: 0});
 
     function handleClick(event: React.MouseEvent<HTMLElement>) {
         event.preventDefault();
-        props.handleArticleChange(props.mapMarker.articleId);
-        props.handleSelectedMapMarkerChange(props.mapMarker);
-        console.log("Click");
+        if(!props.isMovable) props.handleSelectedMapMarkerChange(props.mapMarker);
+    }
+
+    useEffect(() => {
+        const yOffset:number = 34; /* Height of the headline. */
+        setCoordinates({
+            xPosition: props.mapMarker.xPosition + props.offsetWorldMapFrame.xOffset - 0.5 * mapMarkerSize.xSize,
+            yPosition: props.mapMarker.yPosition + props.offsetWorldMapFrame.yOffset - 0.5 * mapMarkerSize.ySize - yOffset
+        });
+        // eslint-disable-next-line
```

Comment:
???

Reply:
I don't want a warning, everytime I change something in MapMarkerCard.tsx (and others), because sonarLint things its best to implement an infinit loop or add useless code like here, where it wants me to change the dependencies to `[mapMarkerSize.xSize, mapMarkerSize.ySize, props]`. Those values are never changed.

Another example would be in App.tsx, where sonarLint wants to change this `useEffect(fetchArticles, [mapMarkers]);` to this `useEffect(fetchArticles, [fetchArticles]);`. I am not even sure what would happens here, since `fetchArticles` is a function.